From: Vagrant Cascadian <vagrant@reproducible-builds.org>
Date: Sun, 30 Apr 2023 15:52:59 -0700
Subject: man/fill_template: Use POSIX compatible comparison.

Bash supports == comparison, but other /bin/sh implementations may
not.

Forwarded: https://github.com/jackaudio/jack2/pull/932
---
 man/fill_template | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/fill_template b/man/fill_template
index 641cab8..63642a3 100644
--- a/man/fill_template
+++ b/man/fill_template
@@ -2,7 +2,7 @@
 d=""
 [ -z "$SOURCE_DATE_EPOCH" ] || d="--utc --date=@$SOURCE_DATE_EPOCH"
 
-if [ "$2" == "True" ]; then
+if [ "$2" = "True" ]; then
   for i in *.0 ; do
     sed -e "s/!VERSION!/${1}/g" -e "s/!DATE!/`date $d '+%Y-%m-%d'`/g" < ${i} > ${i%%0}1
   done
